lib/msf/core/module/options.rb

```diff
@@ -63,7 +61,6 @@ def register_evasion_options(options, owner = self.class)
   #
   def register_options(options, owner = self.class)
     self.options.add_options(options, owner)
-    self.datastore.import_options(self.options, 'self', true)
```

Comment:
I statically verified this, it seemed like `import_defaults(false)` already calls this line under the hood and it seemed good to remove